util/util.go

```diff
@@ -175,6 +175,23 @@ func FindImage(store storage.Store, firstRegistry string, systemContext *types.S
 	return ref, localImage.StorageImage(), nil
 }
 
+// LookupImage returns *Image to corresponding imagename or id
```

Comment:
Move this to a new `internal/util` package, if it needs to exist at all.  Two of its callers call it from a loop, where creating new runtime objects for each iteration is going to be wasteful.

Reply:
Fixed in latest commit.